function-app-container/entrypoint.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/bash
+
+set -e
+
+if [[ -z $WEB_APP_NAME ]];
+then 
+    echo "Required Web App Name. Provide value in WEB_APP_NAME variable" >&2
+    exit 1
+fi
+
+echo "Web App Name: ${WEB_APP_NAME}"
```

Comment:
Is there a notion of System.Debug in actions. If something fails and we would like to get more debug statements, what is the recommendation here.

Reply:
currently, there is no inbuilt debug flag.